src/test_utils.jl

```diff
@@ -0,0 +1,56 @@
+"""
+    FeatureTransforms.TestUtils
+
+Provides fake [`Transform`](@ref)s for testing purposes only.
+"""
+
+module TestUtils
+
+using ..FeatureTransforms
+using ..FeatureTransforms: OneToOne, OneToMany, ManyToOne, ManyToMany
+using Tables
+
+export FakeOneToOneTransform, FakeOneToManyTransform
+export FakeManyToOneTransform, FakeManyToManyTransform
+
+for C in (:OneToOne, :OneToMany, :ManyToOne, :ManyToMany)
+    FT = Symbol(:Fake, C, :Transform)
+    @eval begin
+        """
+            $($FT) <: Transform
+
+        A fake [`$($C)`](@ref) transform for test purposes.
```

Comment:
I think we should say more here on why the fake transforms are named according to cardinality, and why that's useful for testing.